caput/memh5.py

```diff
@@ -2445,13 +2733,24 @@ def _copy_to_file(memgroup, h5group):
                     data = check_unicode(item)
 
                     # Write to file from MPIArray
-                    data.to_hdf5(
-                        h5group,
-                        key,
-                        chunks=item.chunks,
-                        compression=item.compression,
-                        compression_opts=item.compression_opts,
-                    )
+                    if fileformats.HDF5.compression_enabled():
```

Comment:
Same comments as above.

Reply:
Also, how did we avoid this before? I guess we just relied on the user not setting any compression as it would break.